DataFormats/SiStripCluster/src/SiStripApproximateCluster.cc

```diff
@@ -1,7 +1,10 @@
 #include "DataFormats/SiStripCluster/interface/SiStripApproximateCluster.h"
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+#include <algorithm>
```

Comment:
Where is this header being used?

Reply:
I believe it is needed for the std::max call on L17

Reply:
I think std::max comes from cmath, and that algorithm is not needed.

Reply:
According to cppreference `std::max()` comes from `algorithm`
https://en.cppreference.com/w/cpp/algorithm/max